packages/config/config/devices/0x0460/qnsw-001P16.json

```
@@ -63,7 +89,7 @@
 		{
 			"#": "20",
 			"label": "Turn O1 On Automatically With Timer",
```

In the spirit of consistency:
```suggestion
			"label": "O1: Auto-On Timer",
```